DATA/common/setenv_calib.sh

```diff
@@ -72,6 +72,17 @@ if [[ $BEAMTYPE != "cosmic" ]] || [[ $FORCECALIBRATIONS == 1 ]] ; then
 
 fi
 
+echo "CALIB_PRIMVTX_MEANVTX = $CALIB_PRIMVTX_MEANVTX" 1>&2
```

Comment:
All these printouts are ok for now for debugging, but in the long run I'd like to remove them, or only activated them with some env variable like `GEN_TOPO_VERBOSE=1`

Reply:
Yes, sure.